ivy/container/extensions/manipulation.py

```diff
@@ -461,3 +461,139 @@ def static_hstack(
             map_sequences=map_sequences,
             out=out,
         )
+
+    @staticmethod
+    def static_rot90(
+        m: Union[ivy.Container, ivy.Array, ivy.NativeArray],
+        /,
+        *,
+        k: Optional[int] = 1,
+        axes: Optional[Tuple[int, int]] = (0, 1),
+        key_chains: Optional[Union[List[str], Dict[str, str]]] = None,
+        to_apply: bool = True,
+        prune_unapplied: bool = False,
+        map_sequences: bool = False,
+        out: Optional[ivy.Container] = None,
+    ) -> ivy.Container:
+        """
+        ivy.Container static method variant of ivy.rot90.
+        This method simply wraps the function, and so the docstring for
+        ivy.rot90 also applies to this method with minimal changes.
+        Parameters
+        ----------
+        m
+            Input array of two or more dimensions.
+        k
+            Number of times the array is rotated by 90 degrees.
+        axes
+            The array is rotated in the plane defined by the axes. Axes must be
+            different.
+        key_chains
+            The key-chains to apply or not apply the method to. Default is None.
+        to_apply
+            If True, the method will be applied to key_chains, otherwise key_chains
+            will be skipped. Default is True.
+        prune_unapplied
+            Whether to prune key_chains for which the function was not applied.
+            Default is False.
+        map_sequences
+            Whether to also map method to sequences (lists, tuples). Default is False.
+        out
+            optional output container, for writing the result to. It must have a shape
+            that the inputs broadcast to.
+        Returns
+        -------
+        ret
+            Container with a rotated view of m.
+        Examples
+        --------
+        >>> m = ivy.Container(a=ivy.array([[1,2], [3,4]]),\
+                b=ivy.native_array([[1,2,3,4],\
+                                    [7,8,9,10],\
+                                    [3,4,5,6],\
+                                    [11,12,13,14]]))
+        >>> ivy.Container.static_rot90(m)
+        {
+            a: ivy.array([[2, 4],
+                          [1, 3]]),
+            b: (<class ivy.array.array.Array> shape=[4, 4])
```

Comment:
The returned `b` is strange here. I think that an `ivy.Container` should be made up of `ivy.Container` or `ivy.Array.array` instances, not `ivy.Array.native_array`. Could you try and define `b` using `ivy.array(...)` to see if the return is fixed?

Reply:
I've changed the initialization as suggested: `b=ivy.array([[1,2,3,4],\
                                    [7,8,9,10],\
                                    [3,4,5,6],\
                                    [11,12,13,14]]))`
This is the output i'm getting.
![Screenshot from 2022-10-26 23-29-03](https://user-images.githubusercontent.com/30499524/198150663-4a62e32c-4423-46a9-bdbc-22f21414fd0f.png)
I checked the `__repr__` special method in `ContainerBase` class and i found that there is parameter `print_limit` which controls this behaviour. Default value is array size `10` so for any array size having no of elements greated than 10, the output is in this format `(<class ivy.array.array.Array> shape=[4, 4])`. Let me know if i misunderstood anything.

Reply:
Oh, you're right. Thanks for looking into this! Then I think you could just use a smaller `b`, just so the example is obvious.

Reply:
Done